Library/Homebrew/formula_auditor.rb

```diff
@@ -393,29 +359,21 @@ def audit_versioned_keg_only
         end
       end
 
-      return if VERSIONED_KEG_ONLY_ALLOWLIST.include?(formula.name)
+      return if tap_audit_exception :versioned_keg_only_allowlist, formula.name
       return if formula.name.start_with?("adoptopenjdk@")
       return if formula.name.start_with?("gcc@")
```

Comment:
I wonder (in a later PR) if this sort of thing should/could be ported over too.

Reply:
May as well add it here. I think the best way to do it will to add all versioned formulae/casks to the `versioned_keg_only_allowlist.json` file. The only downside that I can think of with that is this won't be as dynamic anymore. We'll need to remote old versioned formulae and add new ones as they're added/removed. The good news is that `brew audit` should complain in either case which will prompt modifying the lists. Does that make sense?

Reply:
This `adoptopenjdk` reference can be removed, right? `adoptopenjdk` is a cask so it isn't checked by `formula_auditor` (plus it doesn't have different versions)

Reply:
yup!